src/current/_includes/v22.2/ui/statements-filter.md

```diff
@@ -10,14 +10,21 @@
 
 The statement fingerprints returned are determined by the selected **Search Criteria**:
 
-1. By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` are returned.
-  - To change the number returned, select `25`, `50`, or `500` from the dropdown.
-  - To change the column filter, select `Contention Time`, `Execution Count`, `P99 Latency`, or `Statement Time` from the dropdown.
+### Search Criteria ###
+
+By default, the **Top** `100` statement fingerprints **By** `% of All Runtime` for the `Past Hour` are returned.
+
+1. To change the top N number returned, select `25`, `50`, `100`, or `500` from the dropdown.
```

Comment:
Should this be "To change the number results returned," instead?

Reply:
Modified to `To change the number of results returned`